assets/ui/taskSelectionScreen.ui

```diff
@@ -1,75 +1,136 @@
+                                    "text" : "Tasks",
+                                    "id"   : "tasksTabButton",
+                                    "family" : "title"
+
+
+                                },
+                                {
+                                    "type" : "UIButton",
+                                    "text" : "Buildings",
+                                    "id"   : "buildingsTabButton",
+                                    "family" : "title"
+
                                 }
-                            ]
+                            ],
+                            "layoutInfo": {
+                                "use-content-height": true,
+                                "position-top": {
+                                  "offset": 32
+                                },
+                                "position-horizontal-center": {
+                                  "offset": -5
+                                },
+                                "width":1000
+                              }
                         },
                         {
                             "type" : "RowLayout",
                             "contents" : [
                                 {
-                                    "type" : "UIButton",
-                                    "id" : "cancelButton",
-                                    "text" : "Cancel Selection"
+                                    "type": "ScrollableArea",
+                                    "content": {
+                                        "type" : "UIList",
+                                        "id" : "taskSelectionScreenList",
+                                        "family" : "titlee"
+                                    }
+                                },
+                                {
+
+                                    "type" : "UILabel",
+                                    "text" : "REFERENCE PIC",
+                                    "id"   : "environmentTaskInfoLabel",
+                                    "family" : "titlee"
+
                                 }
-                            ]
+                            ],
+                             "layoutInfo": {
+                                 "position-top": {
+                                     "offset": 80
+                                     },
+                                 "position-horizontal-center": {
+
+                                 },
+                                 "height": 600,
+                                 "width": 900
+                             }
+                        },
+                        {
+                            "type" : "UIButton",
+                            "id" : "confirmButton",
+                            "text" : "Confirm",
+                            "layoutInfo": {
+                                "use-content-height": true,
+                                "position-bottom": {
```

Comment:
@skaldarnar this code seemed to move it to the bottom. I did manage to place it to all other corners, except the right bottom one.

Reply:
Hm, that's weird. I tried a couple of combinations but ended up with the most obvious one working as intended 🙄 (with `position-horizontal-...` it ended up in weird locations, e.g., on top of the cancel "X" button...).

```json
  "position-right": {
    "offset": 1
  },
  "position-bottom": {
    "offset": 1
  },
```

![image](https://user-images.githubusercontent.com/1448874/83975866-6baad380-a8f6-11ea-8c9e-4e5e2794036c.png)